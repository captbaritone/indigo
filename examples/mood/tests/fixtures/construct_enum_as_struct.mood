enum Foo {
    A,
    B,
    C,
}

pub fn test(): i32 {
  Foo {
    bar: i32
  };

  0_i32
}