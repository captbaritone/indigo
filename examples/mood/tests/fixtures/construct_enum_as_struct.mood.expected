Error: Tried to use a enum as a struct.:
 --> construct_enum_as_struct.mood:8:3

  | pub fn test(): i32 {
8 |   Foo {
  |   ^^^ "Foo" is a enum not a struct.
  |     bar: i32
